src/templates/role.js

```diff
@@ -46,7 +47,7 @@ const CenteredContent = styled.div`
   text-align: center;
 `
 
-const ApplyPage = ({ data, pageContext }) => {
+const RolePage = ({ data, pageContext }) => {
```

Comment:
I didn't change much on the role template, but thoughts on if anything/what should be changed?

Reply:
We should leave "breadcrumbs" to help users return to the apply page once they're on a role page. I feel lost when i get to the role page. 
eg: 
![image](https://user-images.githubusercontent.com/7122182/79783783-2f53f180-830f-11ea-8089-b9be4e86d739.png)

So it could be Apply > Designer where Apply links you back.